pynest/nest/lib/hl_api_info.py

```diff
@@ -66,14 +66,24 @@ def authors():
 
 @check_stack
 def helpdesk():
-    """Open the NEST helpdesk in browser.
+    """Open the NEST documentation index in a browser.
 
-    Use the system default browser.
+    This command opens the NEST documentation index page using the
+    system's default browser.
+
+    Please note that the help pages will only be available if you ran
+    ``make html`` prior to installing NEST. For more details, see
+    :ref:`documentation_workflow`.
 
     """
 
-    nestdocdir = sli_func("statusdict/prgdocdir ::")
-    helpfile = os.path.join(nestdocdir, 'help', 'helpindex.html')
+    docdir = sli_func("statusdict/prgdocdir ::")
+    help_fname = os.path.join(docdir, 'html', 'index.html')
```

Comment:
You changed the filename here to `help_fname`, but on lines 81, 86 and 88 below it is still `helpfile`.

On macOS, the problem mention on line 83f has been fixed long ago, so we no longer need to treat macOS special. On the other hand, on my machine it seems that passing just the filename to `open_new()` has no effect, it has to be a proper URL beginning with `file://`. Documentation for `open_new()` also states that the argument should be a URL (Python 3.8.5).

Reply:
Too many code paths to test... Sorry. Fixed in 5a19331.

I've also added `file://` to the path and as this also seems to work on Linux, I've removed the conditional check on the operating system.